container-storage-setup.sh

```diff
@@ -1005,10 +1011,9 @@ is_valid_storage_driver() {
   # Empty driver is valid. That means user does not want us to setup any
   # storage.
   [ -z "$driver" ] && return 0
-
-  for d in $_STORAGE_DRIVERS;do
-    [ "$driver" == "$d" ] && return 0
-  done
+  # Must be an overlay variant or devicemapper
+  is_overlay_driver "$driver" && return 0
+  test "$driver" == "devicemapper" && return 0
```

Comment:
I think there is no need to change this. Adding another driver does not change code. In fact older code is simpler and does not care what's in the list.

Reply:
Every other case though needs to distinguish between them, it seems simpler to have another `is_overlay_driver` call than to have a list only used by one caller right?

Reply:
we already have a list and if list traversal is easy, I really can't see why should not we do this in this case. In other cases using `is_overlay_driver` is just fine.